tests/test_pe.py

```diff
@@ -105,6 +105,21 @@ def test_mult2():
     assert res_p==0
     assert irq==0
 
+def test_fp_add():
+    pe = PE()
+    inst = asm.fp_add()
+    res, res_p, irq = pe(inst, Data(0x801),Data(0x782))
```

Comment:
"Data(0x801)" is not very descriptive of what the test is trying to do methinks...would it be possible to add a comment something like "# (0x802, 0x782, 0x842) in bfloat is (1.2, 0.8, 2.0)" and/or instead of "Data(0x801)" it could be something like "BFloat(1.2)" (or whatever it is)

Reply:
Note the same comment (above) applies to fp_mul etc. Also this is just a mild suggestion, feel free to ignore if inappropriate/unnecessary...

Reply:
Good point... added some comments to explain